r/R/feather.R

```diff
@@ -144,17 +144,20 @@ write_feather <- function(x,
 #' }
 read_feather <- function(file, col_select = NULL, as_data_frame = TRUE, ...) {
   if (!inherits(file, "RandomAccessFile")) {
-    file <- make_readable_file(file)
+    file <- arrow:::make_readable_file(file)
```

Comment:
What's up with `arrow:::` here? this shouldn't be necessary since we are already in arrow's private namespace, and will cause issues during R CMD check